src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -101,22 +101,43 @@ namespace winrt::TerminalApp::implementation
     // - Moves the focus up or down the list of commands. If we're at the top,
     //   we'll loop around to the bottom, and vice-versa.
     // Arguments:
-    // - moveDown: if true, we're attempting to move to the next item in the
-    //   list. Otherwise, we're attempting to move to the previous.
+    // - moveDown: if true, we're attempting to move to the next or a few next item in the
+    //   list. Otherwise, we're attempting to move to the previous or not very many previous.
     // Return Value:
     // - <none>
-    void CommandPalette::SelectNextItem(const bool moveDown)
+    void CommandPalette::SelectNextItem(const bool moveDown, const bool pageButtonPressed)
     {
+        const auto itemHeight = ::base::saturated_cast<int>(_parentCommandText().Height());
```

Comment:
I wouldn't take a dependency on the height of the `parentCommandText` element, there's not necessarily a guarantee that element is the same height as the items of the list view